devtools/common/src/main/resources/templates/dockerfile-jvm.ftl

```diff
@@ -0,0 +1,19 @@
+####
+# Before building the docker image run:
+#
+# mvn package
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile.jvm -t quarkus/${project_artifactId}-jvm .
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 quarkus/${project_artifactId}-jvm
+#
+###
+FROM fabric8/java-jboss-openjdk8-jdk
```

Comment:
I am not sure if this image is still maintained. Better use `jboss/base-jdk:8` instead?

Reply:
@rhuss any idea?